src/Manager/RepositoryManagerInterface.php

```diff
@@ -25,9 +25,6 @@ interface RepositoryManagerInterface
     /**
      * Adds index name and its finder.
      * Custom repository class name can also be added.
-     *
-     * @param string $indexName      The type name in "index/type" format
-     * @param string $repositoryName
      */
     public function addIndex(string $indexName, FinderInterface $finder, string $repositoryName = null): void;
```

Comment:
```suggestion
    public function addIndex(string $indexName, FinderInterface $finder, ?string $repositoryName = null): void;
```

Reply:
There is a CS rule for that: `nullable_type_declaration_for_default_null_value`

Reply:
I remember adding it to ruflin/elastica ;)

Reply:
Nice. PR has been updated :)